src/helpers/snapshot.ts

```diff
@@ -117,10 +120,15 @@ export async function proposalValidation(
     space.id,
     space.network,
     'latest',
-    // this is needed until we change proposal validation
-    { minScore: space.filters.minScore, strategies: space.strategies },
+    {
+      minScore: space.validation?.params?.minScore || space.filters.minScore,
+      strategies: space.validation?.params?.strategies || space.strategies
+    },
```

Comment:
I just noticed we are not passing other params here, maybe should be like
```suggestion
    {
      minScore: space.validation?.params?.minScore || space.filters.minScore,
      strategies: space.validation?.params?.strategies || space.strategies
      ...space.validation?.params
    },
```

But also we don't use minScore and strategies on passport validation, may these should be optional, not sure how 😄

Reply:
Maybe we can pass minScore and strategies only for basic validation?

Reply:
I mean like here https://github.com/snapshot-labs/snapshot-sequencer/pull/30/files (Didn't test it)

Reply:
Yes, good catch again!

Reply:
How is [this](https://github.com/snapshot-labs/snapshot/pull/3558/commits/6192eff5bdb88364f538ac2ffaf5b6dc18874e6e)?